tests/bluetooth/controller/common/include/helper_features.h

```diff
@@ -111,46 +111,46 @@
 #define FEAT_MIN_CHANN 0x00
 #endif
 
-#if defined(CONFIG_MISSING)
-#define FEAT_CTE_REQ 0x20000
+#if defined(CONFIG_BT_CTLR_DF_CONN_CTE_REQ)
+#define FEAT_CONN_CTE_REQ 0x20000
 #else
-#define FEAT_CTE_REQ 0x00
+#define FEAT_CONN_CTE_REQ 0x00
 #endif
 
-#if defined(CONFIG_MISSING)
-#define FEAT_CTE_RSP 0x40000
+#if defined(CONFIG_BT_CTLR_DF_CONN_CTE_RSP)
+#define FEAT_CONN_CTE_RSP 0x40000
 #else
-#define FEAT_CTE_RSP 0x00
+#define FEAT_CONN_CTE_RSP 0x00
 #endif
 
 #if defined(CONFIG_MISSING)
-#define FEAT_CTE_TX 0x80000
+#define FEAT_CONNECTIONLESS_CTE_TX 0x80000
```

Comment:
Is it on purpose that this #define is still under #if defined(CONFIG_MISSING)?

Reply:
Yes, this feature is irrelevant for LLCP. It is a feature of CTE that works only with periodic advertising. There is no requirement to enable it for LLCP tests, similar to `FEAT_PER_ADV_SYNC_TX` and `FEAT_PER_ADV_SYNC_RX`